css/main.css

```diff
@@ -1,3 +1,263 @@
-/* Add your CSS here */
+*:root {
+  --main-bg-color: #fff;
+  --primary: #e6dacd; /*dark beige*/
+  --secondary: #f4ece6; /*light beige*/
+  --accent-colour: #0150fd; /*royal beige*/
+  --body-color: #000;
+}
 
-/* Dont' forget to link this file to your HTML in the <head> */
+body {
+  font-family: "Quicksand", sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+.square-h1 {
+  display: flex;
+  flex-direction: row;
+  gap: 5px;
+}
+
+.homepage-main {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: 126.4px 952px;
+
+  grid-template-areas:
+    "header header "
+    "content  aside"
+    "footer footer";
+}
+
+.homepage-header {
+  background-color: var(--main-bg-color);
+  grid-column-end: -1;
+  grid-column-start: 1;
+  grid-row: 1;
+}
+
+.homepage-content {
+  background-color: var(--secondary);
+  margin-left: 10px;
+  grid-column-start: 1;
+  grid-column-end: -1;
+  padding: 10px;
+  width: 100vw;
+  margin: 20px;
+  width: 280px;
+  height: 375px;
```

Comment:
we shouldn't be using height on this, as this is counter-intuitive. As soon as you add a bit of texts, or resize texts, then you need to change the height again.

Removing height will make the section to have a background colour, therefore you need to split the section into two to prevent this from happening (i.e. the Maya Nelson box and the Hello text box)